packages/salesforcedx-vscode-core/test/vscode-integration/commands/util/retrieveParser.test.ts

```diff
@@ -10,6 +10,11 @@ import {
   registryData,
   SourceComponent
 } from '@salesforce/source-deploy-retrieve';
+import {
+  ComponentRetrieval,
+  RetrieveStatus,
+  SourceRetrieveResult
+} from '@salesforce/source-deploy-retrieve/lib/src/client/types';
```

Comment:
Not for this PR but these types should be surfaced by just importing from `@salesforce/source-deploy-retrieve`

Reply:
Yeah we forgot to export them earlier 😢  Roy opened up this [WI](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008hEKCIA2/view) to address it